pkg/recommender/engine.go

```diff
@@ -258,6 +263,32 @@ func (e *Engine) RecommendCluster(ctx context.Context, provider string, service
 	}, nil
 }
 
+func (e *Engine) getControlPlane(provider, service, region string, cheapestNodePoolSet []NodePool) ([]NodePool, error) {
```

Comment:
please rewrite this method to return the control plane only and append it to the nodepools if it is the case